### DISCUSSION TITLE
怎么改头像

### DISCUSSION THREAD

**Discussion by CBBC123** [share121/inter-knot#2717] [其他]
我想换个头像🙃
![1723642233411](https://github.com/user-attachments/assets/d917dfc4-75d0-448a-ac57-554928594dc3)

**Comment by jian-li-5205** [comment]
不会

**Comment by cyberrobber** [comment]
点头像不就能改吗

**Comment by ysd1911** [comment]
进你的GitHub里的个人主页里可以换，这里的头像就是GitHub的头像<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by jfyygt666** [comment]
不会
